Desktop/modules/upgrader/changecopy.cpp

```diff
@@ -24,6 +24,9 @@ void ChangeCopy::applyUpgrade(Json::Value & options, UpgradeMsgs & msgs) const
 	msgs[newName]		= msgs[oldName];
 
 	msgs[logId].push_back(prefixLog + "Copied option '" + oldName + "' to '" + newName + "'");
+
+	if(msg() != "")
```

Comment:
Is there a reason why you're not doing `msg().isEmpty()`?

Reply:
save like 4 characters at least and I like the look of it ^^